src/lib/firebaseBackend.js

```diff
@@ -61,6 +61,9 @@ class FirebaseBackend extends BackendInterface {
         .collection('dropSite')
         .doc(dropSiteId)
         .get();
+      if (!doc.exists) {
+        return false;
```

Comment:
If we still think this is necessary we'll need to reject the promise.

Reply:
I think rejecting the promise would be more for an error. In this case it's returning that there are no dropSites for this facility.